PlexRequests.Services/Interfaces/INotification.cs

```diff
@@ -35,5 +36,7 @@ public interface INotification
         string NotificationName { get; }
 
         Task NotifyAsync(NotificationModel model);
+
+        Task NotifyAsync(NotificationModel model, Settings settings);

```

Comment:
Discussed in gitter chat -- needed for passing in test settings before a save is actually committed.